tests/Hedgehog.Linq.Tests/LinqTests.cs

```diff
@@ -1,16 +1,38 @@
-﻿using Xunit;
+﻿using System;
+using Xunit;
 
 // Import ForAll:
 using static Hedgehog.Linq.Property;
 
 namespace Hedgehog.Linq.Tests
 {
-    /*
-     * The main object here is just to make sure that the examples compile,
-     * there's nothing fancy in the properties being tested.
-     */
-    public class LinqTests
-    {
+    public class LinqTests {
+
+        [Fact]
+        public void ExceptionInSelect_Action_FailedStatus() {
+            void action() => throw new Exception();
+            var property =
+                from _ in Property.ForAll(Gen.Int32(Range.Constant(0, 0)))
+                select action();
+            var report = property.Report();
+            Assert.True(report.Status.IsFailed);
+        }
+
+        [Fact]
+        public void ExceptionInSelect_Func_FailedStatus() {
+            bool func() => throw new Exception();
+            var property =
+                from x in Property.ForAll(Gen.Int32(Range.Constant(0, 0)))
+                select func();
+            var report = property.Report();
+            Assert.True(report.Status.IsFailed);
+        }
+
+        /*
+         * The main object the following tests is just to make sure that the examples compile.
+         * There's nothing fancy in the properties being tested.
+         */
+
```

Comment:
Code style here doesn't match the rest of the code, the braces should be on a separate line.

Reply:
Done